### PR TITLE
Added New Js Question

### PR DESCRIPTION
**Pull request by shubhamr238**
Added New JavaScript Question: Difference between “var” and “let” Keywords?

### CODE REVIEW

**Comment by shivamyth** [comment]
Any update on this. We will close the PR in next 24 hours if there will no updates

**Comment by shubhamr238** [comment]
what to update plz specify.

File: javascript/index.html [hunk 1]
```diff
@@ -111,7 +111,11 @@ <h4>
             <li>
                 <a href="./answers/q5.html"><h4>What are escape Sequences?</h4></a>
             </li>
-
+            <li>
+                <a href="./answers/var_vs_let.html">
```
Comment (shivamyth) [inline javascript/index.html]: change the name of ans html file as the format done by others check that eg:{name}-q{no.}

Comment (shubhamr238) [reply javascript/index.html]: done.

**Comment by shubhamr238** [comment]
updated as requested.

**Comment by shubhamr238** [comment]
kindly merge my PR for question no 7

**Review by arpansac — APPROVED** [review]
Looks good!

**Pull request merged**